### PR TITLE
error when using non-default dataset/stations dim names from cli

### PR DESCRIPTION
**Pull request by colonesej**
Fix error when providing non-default options via cli config file. 
Current extract only using default config values

### CODE REVIEW

**Comment by FussyDuck** [comment]
[CLA assistant check](https://bol-claassistant.ecmwf.int:443/ecmwf/hat?pullRequest=26) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://bol-claassistant.ecmwf.int:443/ecmwf/hat?pullRequest=26) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://bol-claassistant.ecmwf.int:443/check/ecmwf/hat?pullRequest=26) it.</sub>

File: hat/geo.py [hunk 1]
```diff
@@ -347,10 +347,14 @@ def get_latlon_keys(ds):
     return lat_key, lon_key
 
 
-def latlon_coords(ds):
+def latlon_coords(ds, names: list = []):
```
Comment (corentincarton) [inline hat/geo.py:350]: Maybe dims instead?

Comment (colonesej) [reply hat/geo.py:350]: actually, this was correct. I mixed it up the netcdf dim names and station coordinates. I can revert it, or just support another optional input about the dim names. What do you think?

Comment (corentincarton) [reply hat/geo.py:350]: Ah, I would revert it then!

**Comment by corentincarton** [comment]
Merged. Thanks @colonesej !

**Pull request merged**